comment_scan.rb

```diff
@@ -440,6 +440,30 @@ def restart(num_to_post, bundle)
         say reasons.join("\n")
       end
     end
+    command "!!/regexstats" do |bot, reason|
+      if matches_bot(bot)
+        #Build array of hashes for each regex containing info to build the stat output
+        regexes = (reason.nil? ? Reason.all : Reason.where(name: reason)).map do |r|
```

Comment:
Instead of `Reason.where(name: reason)` you may want to do whatever the sqlite equivalent of `LIKE` is. Or if you just want one reason, `[Reason.find_by(name: reason)]` instead of `where`.

Reply:
Ooh super clever on the `LIKE`. I didn't even think of that. I was just blindly copying what we do with `!!/regexes`, but that's way better. Good call.

Reply:
I mean, this change may also make sense for `!!/regexes`, feel free to add it if you think it makes sense.